src/react-apps/applications/ux-editor/src/sagas/formDesigner/formDesignerSagas.ts

```diff
@@ -267,8 +267,11 @@ function* generateRepeatingGroupsSaga({ }: FormDesignerActions.IGenerateRepeatin
         continue;
       }
 
-      const repeatingData = Object.keys(formFillerState.formData).filter((formDataKey) => {
-        return formDataKey.includes(container.dataModelGroup + '[');
+      const repeatingData = Object.keys(formFillerState.formData).filter((formDataKey: any) => {
+        const match = formDataKey.indexOf(container.dataModelGroup + '[') > -1;
+        if (match) {
+          return formDataKey;
```

Comment:
The earlier code (before change) returned true if it contains `${container.dataModelGroup}[`.
Now it returns the key of the `formData`. Changed to this deliberately?